lnwire/announcement_signatures_2.go

```diff
@@ -0,0 +1,97 @@
+package lnwire
+
+import (
+	"bytes"
+	"io"
+)
+
+// AnnounceSignatures2 is a direct message between two endpoints of a
```

Comment:
maybe mention who sends 1 and who sends 2? Atm the doc is just a dup of `AnnounceSignatures1`